gateway/src/main/java/io/zeebe/gateway/metrics/GatewayMetrics.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.gateway.metrics;
+
+import io.prometheus.client.Counter;
+import io.prometheus.client.Histogram;
+
+public final class GatewayMetrics {
+
+  private static final Histogram REQUEST_LATENCY =
+      Histogram.build()
+          .namespace("zeebe")
+          .name("gateway_request_latency")
+          .help("Latency of round-trip from gateway to broker")
+          .labelNames("partition", "requestType")
+          .register();
+
+  private static final Counter FAILED_REQUESTS =
```

Comment:
May be we can add the reasons for failure to the labels. We can then distinguish timeouts vs other failures.